src/connection/status.js

```diff
@@ -1,7 +1,12 @@
 'use strict'
 
-module.exports = {
+const STATUS = {
```

Comment:
This seems to be causing issues integrating with libp2p PR

Reply:
"Property 'OPEN' does not exist on type 'typeof import("/Users/vsantos/work/pl/gh/libp2p/js-libp2p-interfaces/dist/src/connection/status")'"

Using: 
```js
const ConnectionStatus = require('libp2p-interfaces/src/connection/status')

ConnectionStatus.OPEN
```